index.bs

```diff
@@ -257,28 +264,112 @@ spec:css-syntax-3;
   This document depends on the Infra Standard for a number of foundational concepts used in its
   algorithms and prose [[!INFRA]].
 
+
+  ### Infrastructure Algorithms ### {#sctn-infra-algorithms}
+
+  <!-- NOTE: '<h4>' maps to the same heading level as three hashs '###', thus using '<h5>' for the next section -->
+  <h5 id="algorithm-same-origin-with-ancestors" algorithm>Same-Origin with its Ancestors</h5>
+
   An [=environment settings object=] (|settings|) is <dfn noexport>same-origin with its
   ancestors</dfn> if the following algorithm returns `true`:
 
-  1.  If |settings| has no [=environment settings object/responsible document=],
-      return `false`.
+  <ol class="algorithm">
+    1.  If |settings| has no [=environment settings object/responsible document=],
+        return `false`.
```

Comment:
well, uh, hm.... respectfully: that strikes me as a "hack" for a very limited purpose.  All the existing algs encompassed by `<ol class="algorithm">` are indented relative to the latter element --- I'm just trying to follow the existing conventions in this spec's source. Thus I'd (please) prefer to not do this. 

(friendly aside: there's various aspects of this spec's source's formatting that I'd like to change (eg  I'd like to eliminate the default 2 space indent of all the source (eg make this spec's source more like webauthn's) and use 4 space indents throughout (that'd be more friendly to my editor settings) but I haven't brought any of this up with my co-editor because it's of only 2d- or 3d-order concern...)

Reply:
Works for me.